cg/cli/add.py

```diff
@@ -359,6 +359,8 @@ def download_external_delivery_data_to_hpc(context: CGConfig, ticket: str, dry_r
 )
 @click.pass_obj
 def add_external_data_to_hk(context: CGConfig, ticket: str, dry_run: bool, force):
-    """Adds external data to Housekeeper"""
-    external_data_api = ExternalDataAPI(config=context)
-    external_data_api.add_transfer_to_housekeeper(dry_run=dry_run, ticket=ticket, force=force)
+    """Adds external data to Housekeeper and starts their analyses."""
+    external_data_api = AddExternalDataAPI(
```

Comment:
Pass via context

Reply:
I checked and MetaAPIs are not passed by context but instantiated on site when the command that requires them is called, so for now I will leave this as it is